amethyst_animation/src/lib.rs

```diff
@@ -65,9 +66,6 @@ extern crate num_traits;
 #[macro_use]
 extern crate serde;
 
-#[cfg(feature = "profiler")]
-extern crate thread_profiler;
```

Comment:
I probably missed a conversation somewhere, but why did you remove this?

Reply:
No you didn't miss anything, I'm just trying to make the compiler happy and it said to get rid of it because it wasn't used.  That being said I'm currently battling with CI and my own compiler on how exactly thread_profiler should be brought in.

Reply:
So are we gonna remove the profiling or what are you planning?

Reply:
The profiling simply wasn't in use for these crates.  I'll add it back in another PR but I considered that out of scope for this one.